### ISSUE TITLE
AreTomo is failing during  write_relion_tilt_series_alignment_output

### ISSUE THREAD

**Issue by EuanPyle** [alisterburt/tomo_preprocessing#23]
Not writing out a star file and failing

**Comment by EuanPyle** [comment]
yep, trying a few things now. May be due to these lines. https://github.com/alisterburt/tomo_preprocessing/blob/918a8475717d44a2e847ac87c7c6248461c8935b/tomography_preprocessing/tilt_series_alignment/aretomo/_utils.py#L66-L67
 I just removed them and it started working. Want to test one more condition without these lines. Think we were going to get rid of them anyway!

**Comment by alisterburt** [comment]
cool, we shouldn't need these any more with the dark tolerance argument anyway :)
